### ISSUE TITLE
wav: Decode: header size can be more than 44 bytes

### ISSUE THREAD

**Issue by faiface** [faiface/beep#2]
`wav.Decode` function assumes that the WAVE header is 44 bytes long. Although this is usually true, it's not always. Header size is determinable by looking at the chunk sizes.

**Comment by mewmew** [comment]
This could be fixed at the same time as adding support for additional chunk types, thus adding support for decoding the example WAV file in https://github.com/faiface/beep/issues/11#issuecomment-353342378

**Comment by faiface** [comment]
Yeah, that's correct. The current WAV decoder is pretty simplistic. Wanna give it a shot? ;)

**Comment by mewmew** [comment]
Hehe, probably easy to implement. Just have to find the right specification. Personally, I'm trying for exams currently, so if there happen to come a PR from me, that means I'm procrastinating ;)

**Comment by faiface** [comment]
This issue was resolved by PR #31.
